pygmt/figure.py

```diff
@@ -306,7 +306,7 @@ def savefig(
 
         fname = Path(fname)
         prefix, suffix = fname.with_suffix("").as_posix(), fname.suffix
-        ext = suffix[1:]  # Remove the .
+        ext = suffix[1:].lower()  # Remove the . and normalize to lowercase
```

Comment:
I don't update the docstrings to document this new enhancement because I don't want to encourage users to use uppercase file extensions.

Reply:
I agree that the use of uppercase file extensions should be avoided. Thus, I feel it is fine to not explicitly mention this possibility in the documentation.